build.gradle

```diff
@@ -92,12 +92,6 @@ allprojects {
     }
   }
 
-  tasks.withType(Javadoc) {
-    // disable the crazy super-strict doclint tool in Java 8
```

Comment:
How about keeping those comments?

Reply:
It seemed a bit redundant since we only support Java 8+. I can add back something.